base/applications/rapps/appview.cpp

```diff
@@ -1524,7 +1524,7 @@ BOOL CApplicationView::ProcessWindowMessage(HWND hwnd, UINT message, WPARAM wPar
     case WM_NOTIFY:
     {
         LPNMHDR pNotifyHeader = (LPNMHDR)lParam;
-        if (pNotifyHeader->hwndFrom == m_ListView->GetWindow())
+        if (pNotifyHeader->hwndFrom == m_ListView->m_hWnd)
```

Comment:
```suggestion
        if (pNotifyHeader->hwndFrom == m_ListView)
```

Reply:
This suggestion doesn't work (I tested it)